packages/editor/src/components/rich-text/index.native.js

```diff
@@ -35,6 +43,14 @@ export class RichText extends Component {
 		this.lastEventCount = 0;
 	}
 
+	onActiveFormatsChange( formats ) {
+		const newFormats = formats.reduce( ( accFormats, activeFormat ) => {
+			accFormats[ activeFormat ] = getFormatValue( activeFormat );
+			return accFormats;
+		}, {} );
+
+		this.setState( { newFormats, selectedNodeId: this.state.selectedNodeId + 1 } );
```

Comment:
Wonder why it's needed `selectedNodeId: this.state.selectedNodeId + 1`  here.

Reply:
copy pasted from the GB web... @gziolo you know why is this?